tests/test_managed_group.rs

```diff
@@ -175,7 +175,7 @@ fn managed_group_operations() {
             .with_invalid_message_received(invalid_message_received)
             .with_error_occured(error_occured);
         let managed_group_config =
-            ManagedGroupConfig::new(HandshakeMessageFormat::Plaintext, update_policy, callbacks);
+            ManagedGroupConfig::new(HandshakeMessageFormat::Ciphertext, update_policy, callbacks);
```

Comment:
Do we still have a test for `Plaintext` after this?

Reply:
Not explicitly, but since all `MLSCiphertext` get converted to `MLSPlaintext` and vice-versa, `MLSPlaintext` are still covered that way.
In fact, this is actually the preferred way, because in this case the `MLSPlaintext`s also get serialized. If we just test with `MLSPlaintext` and ommit the serialization out of convenience (as it was the case), we don't trigger the validity checks. This is how I discovered that the signature was missing.

Reply:
Sure, but either the control flow differs depending on this value, then we need a test for this, or it doesn't, then we don't need the value in the first place.